solver.rb

```diff
@@ -0,0 +1,22 @@
+class Solver
+  def factorial(number)
+    raise 'Please enter positive number' if number.negative?
+    return 1 if number.zero?
+
+    number * factorial(number - 1)
+  end
+
+  def reverse(word)
+    word.reverse
+  end
+
+  def fizzbuzz(number)
+    solver = ''
+
+    solver += 'fizz' if (number % 3).zero?
+    solver += 'buzz' if (number % 5).zero?
+    return number if solver.strip.empty?
+
```

Comment:
- [ ] It should return the string version of the number if it is not divisible by 3 or 5. Currently, it returns the number as it is. Kindly fix this. 👍

Reply:
@Gambit142 Thanks for reviewing it, We fix the issues.

Reply:
fixed it @Gambit142 👍🏼